src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultObjectValidator.cs

```diff
@@ -14,34 +15,36 @@ public class DefaultObjectValidator : IObjectModelValidator
     {
         private readonly IModelMetadataProvider _modelMetadataProvider;
         private readonly ValidatorCache _validatorCache;
+        private readonly IModelValidatorProvider _validatorProvider;
 
         /// <summary>
         /// Initializes a new instance of <see cref="DefaultObjectValidator"/>.
         /// </summary>
         /// <param name="modelMetadataProvider">The <see cref="IModelMetadataProvider"/>.</param>
-        /// <param name="validatorCache">The <see cref="ValidatorCache"/>.</param>
```

Comment:
Strange to take the `ValidatorCache` out of DI. Why does the `DefaultObjectValidator` now own the instance?